platform/machine/aws/machine.go

```diff
@@ -113,7 +113,12 @@ func (am *machine) saveConsole(origConsole string) error {
 		return true, nil
 	})
 	if err != nil {
-		return fmt.Errorf("retrieving console output of %v: %v", am.ID(), err)
+		err = fmt.Errorf("retrieving console output of %v: %v", am.ID(), err)
+		if origConsole != "" {
+			plog.Warning(err)
```

Comment:
Actually, I'd drop this to debug level. It's not really a useful thing to see most of the time unless debugging

Reply:
It is an actual (non-fatal) error, and under other circumstances (tests that run in less than ~5 minutes) will be logged that way by the caller.